src/Popover/Popover.tsx

```diff
@@ -377,45 +448,45 @@ export default class Popover extends React.Component {
   };
 
   getVisiblePopover = () => {
-    const { popover } = this.state;
+    const popover = this.state.popover!; // TODO(gabro): is this safe?
```

Comment:
this doesn't change the current behavior, so it's probably safe, still I would like to double check to see whether we have potential bugs.

Reply:
this is how it works:
1. is `this.initialized` is `false` we render `getHiddenPopover` which doesn't use any state variable
2. `state.popover` and `state.child` are created/updated by `saveValuesFromNodeTree` in `componentDidMount` and `componentWillReceiveProps` (so they should exist after the first `render`)
3. in `componentDidMount` we set `this.initialized` to `true`
4. from here `state.popover` and `state.child` should exist and can be used by `getVisiblePopover`

there may be some edge cases where this process fail, but we haven't had a bug since a lot of time. Still, i'm open to improvements :)

Reply:
ok! I'll remove the comments